Assets/Scripts/SS3D/Systems/Entities/Entity.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FishNet.Object;
+using SS3D.Systems.Inventory;
+using UnityEngine;
+
+namespace SS3D.Systems.Entities
+{
+    /// <summary>
+    /// Base class for all things that can be controlled by a player
+    /// </summary>
+    public class Entity : NetworkBehaviour
```

Comment:
I don't see the point in having an Entity component. Unity already has game objects, so you can just check if an object has the required components to be controlled by the player.